Notebooks/GPRWidget.ipynb

```diff
@@ -114,15 +134,26 @@
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "1178a2f5eb464c89a2f963fc1e2efee3"
+       "model_id": "99f66c82ed65437795819826bcb89db3"
       }
      },
      "metadata": {},
      "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<function gpgLabs.GPR.GPRlab1.PrimaryFieldWidgetFcn>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "ifield = interact(PrimaryFieldWidget,  tinterp = (0, 80, 2), epsr = (1, 40, 1))"
+    "radargramImage = \"../figures/GPR/ubc_GPRcmp.png\"\n",
+    "PrimaryFieldWidget(radargramImage)"
```

Comment:
@dccowan : Fix the window size (ax.set_xlim(), ax.set_ylim())
The window size change with the widget, which it should not as we are looking at 'fix' data